### ISSUE TITLE
Release v0.9

### ISSUE THREAD

**Issue by marianschmidt** [marianschmidt/msSPChelpR#53]
Solve urgent issues:
- [x] issues for milestone are solved

Prepare for release:

- [x]  `devtools::check()`
- [x]  `devtools::spell_check()`
- [x]  `rhub::check_for_cran()` -> rhub::check_for_cran(env_vars=c(R_COMPILE_AND_INSTALL_PACKAGES = "always"))
- [x]  `rhub::check(platform = 'ubuntu-rchk')`
- [x]  `devtools::check_win_devel()`
- [x]  Update `cran-comments.md`
- [x]  [Polish NEWS](https://style.tidyverse.org/news.html#news-release)

Submit to CRAN:

- [x]  Update version # in DESCRIPTION or `usethis::use_version('minor')`
- [x]  Remove "in development" tag from NEWS or `usethis::use_version('minor')`
- [x]  `devtools::release()`
- [x]  Approve email
 
Wait for CRAN...

- [x]  Accepted 🎉
- [x]  `usethis::use_github_release()`
- [ ]  `usethis::use_dev_version()`
- [ ] Tag release on Github

**Comment by marianschmidt** [comment]
Code for copy paste.

```
devtools::check()
devtools::spell_check()
devtools::check_win_devel()
rhub::check_for_cran(env_vars=c(R_COMPILE_AND_INSTALL_PACKAGES = "always"))
rhub::check(platform = 'ubuntu-rchk')
```

**Comment by marianschmidt** [comment]
closed via #87
